app/services/analytics_events.rb

```diff
@@ -1101,7 +1101,7 @@ def idv_enter_password_complete(
     **extra
   )
     track_event(
-      'IdV: review complete', # rubocop:disable IdentityIdp/AnalyticsEventNameLinter
+      'IdV: enter password submitted', # rubocop:disable IdentityIdp/AnalyticsEventNameLinter
```

Comment:
Since we're changing these events, should we move to the [new naming scheme](https://github.com/18F/identity-idp/pull/9318)?
```suggestion
      :idv_enter_password_submitted,
```

Reply:
This will also now fail if merged to `main` as-is, since we've since merged stronger enforcement of the new naming scheme. This branch will need to be rebased and the event renamed to the new scheme.

Reply:
Yup! Done in b22a2012f